src/main/java/org/mule/service/http/impl/service/server/grizzly/GrizzlyServerManager.java

```diff
@@ -65,7 +65,7 @@
   public static final String MAXIMUM_HEADER_SECTION_SIZE_PROPERTY_KEY = SYSTEM_PROPERTY_PREFIX + "http.headerSectionSize";
   private static final int MAX_KEEP_ALIVE_REQUESTS = -1;
   private static final int MIN_SELECTORS_FOR_DEDICATED_ACCEPTOR =
-      getInteger(GrizzlyServerManager.class.getName() + ".MIN_SELECTORS_FOR_DEDICATED_ACCEPTOR", 8);
+      getInteger(GrizzlyServerManager.class.getName() + ".MIN_SELECTORS_FOR_DEDICATED_ACCEPTOR", 4);
```

Comment:
why this change?

Reply:
Is 4 the smallest possible number thats works fine? I'd prefer this value to be a function of the cores, and the minimum value possible. If a greater value is needed, the system property is set.